jina/serve/networking.py

```diff
@@ -138,7 +156,10 @@ def _create_connection(self, address):
         use_tls = parsed_address.scheme in TLS_PROTOCOL_SCHEMES
 
         stubs, channel = GrpcConnectionPool.create_async_channel_stub(
-            address, tls=use_tls, summary=self.summary
+            address,
+            metrics=self._metrics,
+            runtime_name=self.runtime_name,
+            tls=use_tls,
```

Comment:
```suggestion
            address, self._metrics, self.runtime_name, tls=use_tls
```
unless they are not really required parameters and in this case let's make them keyword arguments in the signature

Reply:
but having them as named argument is just more powerful than having them as keyword right ? And it does behave like keyword arugments as well. I am not sure to get why it would be better